tests/mocks.py

```diff
@@ -53,7 +53,7 @@ def _generate_fake_scores(self, batch: torch.LongTensor) -> torch.FloatTensor:
         return batch_scores
 
     def score_hrt(self, hrt_batch: torch.LongTensor) -> torch.FloatTensor:  # noqa: D102
-        return self._generate_fake_scores(batch=hrt_batch)
+        return self.scores[torch.randint(high=self.num_entities, size=hrt_batch.shape[:-1])]
```

Comment:
@cthoyt the error actually came from here 😅

Reply:
woah I wasn't looking anywhere near there. Have the CI trigger when you're confident that's all we need to fix